packages/auth/src/signature/middleware.test.ts

```diff
@@ -375,17 +328,16 @@ describe('Signature Service', (): void => {
           url: tokenManagementUrl,
           method
         },
-        { managementId }
+        { id: managementId },
+        deps
       )
 
       ctx.request.body = {
         access_token: token.value,
         proof: 'httpsig',
         resource_server: 'test'
       }
-      await signatureService.tokenHttpsigMiddleware(ctx, next)
-
-      expect(next).toHaveBeenCalled()
+      await tokenHttpsigMiddleware(ctx, next)
       expect(ctx.response.status).toEqual(400)
```

Comment:
This could check the message, and tests could be added for the other possible errors